.github/workflows/ci.yml

```diff
@@ -12,6 +12,8 @@ on:
 jobs:
   build:
     name: 'build'
+    # CUE Central Registry login does not work on forks because GitHub Secret is not available
+    if: ${{ ! github.event.pull_request.head.repo.fork }}
```

Comment:
yeah but the issue by doing that, is that we don't know anymore if the plugin can be built

Reply:
that's not true, we changed the build of the plugin and now the CLI is running the frontend build

Reply:
but yes the tests for the frontend are still running that is true

Reply:
hm, yes, iirc `jest` compiles just-in-time, i.e. only files which are required by tests. then we only have type-check left :\

so, what should we do? Is the CUE registry dropping the requirement of needing a token anytime soon?

Reply:
we should have an answer this week, but for the moment I have no good solution :(

Reply:
or maybe we could cache the cue dependencies? we don't need to download them every time from the central registry. of course this breaks when a fork is updating the dependencies, but I think that won't happen as much

Reply:
ah yeah that's an idea that will mitigate the dependency from the Cuelang team decision.